packages/@react-native-community/changelog-generator/src/__tests__/changelog-generator.test.ts

```diff
@@ -39,7 +38,7 @@ function requestWithFixtureResponse(fixture: string) {
   (responseEmitter as any).headers = { link: 'rel="next"' };
   setImmediate(() => {
     requestEmitter.emit("response", responseEmitter);
-    readFile(path.join(__dirname, "__fixtures__", fixture), "utf-8").then(
+    void readFile(path.join(__dirname, "__fixtures__", fixture), "utf-8").then(
```

Comment:
Explicitly telling `no-missing-promises` rule that we do not need to handle the returned promise.